### ISSUE TITLE
Add other socials links 

### ISSUE THREAD

**Issue by AJS23** [opentffoundation/manifesto#421]
I guess adding links of opentf  to other social media platforms will help to broadcast the message on a wider range

**Comment by nemobis** [comment]
Yes, at very least something in the Fediverse. It's a bit weird to have only proprietary software communication options despite the proposed goal of being "Truly open source".

**Comment by Dennis1025-dev** [comment]
Do you have other social links
